plugin-build/src/main/kotlin/io/sentry/android/gradle/util/ReleaseInfo.kt

```diff
@@ -0,0 +1,7 @@
+package io.sentry.android.gradle.util
+
+data class ReleaseInfo(
+    val applicationId: String,
+    val versionName: String,
+    val versionCode: Int? = null
+)
```

Comment:
```suggestion
) : Serializable
```
`m`: I think it has to be serializable if we pass it as a task input